compiler/AST/checkAST.cpp

```diff
@@ -247,6 +247,7 @@ void checkPrimitives()
      case PRIM_CAPTURE_FN:
      case PRIM_CREATE_FN_TYPE:
      case PRIM_STRING_COPY:
+     case PRIM_MEMCPY:
```

Comment:
Could we extend `gatherWellKnownTypes` in `filesToAST.cpp` and make it into gatherWellKnownAST or something similar?